gs-rest-server/src/main/java/server/FixMyBugController.java

```diff
@@ -43,4 +45,30 @@ public DatabaseEntry clientCode(@RequestBody String input) {
 
         return new DatabaseEntry(-1, -2, "crap", "squid", -5);
     }
+
+    @RequestMapping("/test")
```

Comment:
If we specify test as our method on the client side, this code (which echos any serverrequest) will run instead of hitting our database.